client/src/modules/app/AppRebuild.tsx

```diff
@@ -69,7 +71,10 @@ export const AppRebuild = ({ appId }: AppRebuildProps) => {
       setIsTerminalVisible(true);
       setRebuildLoading(true);
     } catch (e) {
-      toast.error(e.message);
+      toast({
```

Comment:
@Akirtovskis what do you think about creating our own component for the toast that extend from the chakra one?

I quite like the API of using `toast.error(e.message);` or `toast.success(e.message);` so if we can keep it that would be super cool :)

Reply:
@pradel sounds good, will have a go at this. And definitely agree that toastify way looks neater.